File: exo_03.rb
print "Quel est ton age ?\n> "
age = gets.chomp
if age.length.zero?
  print "Et ton age ?\n"
else
  delta = Time.new.year - 2017
  prevage = age.to_i - delta
  if prevage <= 0
    print "Tu ne peux pas être aussi jeune ?\n"
  else
    print "Tu avais au moins #{prevage} ans en 2017!\n"
  end
end

File: ex/exo_03.exs
age = IO.gets("Quel est ton age ?\n> ") |> String.trim("\n")

cond do
  String.length(age) == 0 ->
    IO.puts("Et ton age ?\n")

  true ->
    date = DateTime.utc_now() |> Map.fetch!(:year)
    delta = date - 2017
    prevage = String.to_integer(age) - delta

    cond do
      prevage <= 0 ->
        IO.puts("Tu ne peux pas être aussi jeune ?\n")

      true ->
        IO.puts("Tu avais au moins #{prevage} ans en 2017!\n")
    end
end
